hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2499,4 +2505,21 @@
       connections.
     </description>
   </property>
+  <property>
+    <name>ozone.csi.default-volume-size</name>
```

Comment:
shall we use the new config annotations instead?

Reply:
Absolutely. I uploaded a new version with config annotations.